gcn_classic_to_json/notices/CALET_GBM_FLT_LC/__init__.py

```diff
@@ -0,0 +1,40 @@
+from ... import utils
+import numpy as np
+
+# Will have to rename schema after the location is decided
+
+
+def parse(bin_arr):
+    # e_high and e_low are both encoded in the same 4 byte integer (as 2 byte shorts)
+    energy_range_data = bin_arr[17:18].view(">i2")
+    # similarly we do this with lat and long
+    lat_lon_data = np.flip(bin_arr[16:17].view(">i2"))
+    lat, lon = lat_lon_data[1], lat_lon_data[0]
+    # Using np.unpackbits to get detector status from bits
+    # I have defined 0 in the binary as "on" and 1 as "triggered"
+    detector_options = ["on", "triggered"]
+    # First convert to an array of 4 1 byte integer values and then convert it into an array of bits
+    # Reversing array so 0th bit is 0-indexed
+    detectors_bits = np.flip(np.unpackbits(bin_arr[19:20].view(dtype="u1")))[:3]
+    detectors_status = [detector_options[bit] for bit in detectors_bits]
+    detectors = dict(zip(["HXM1", "HMX2", "SGM"], detectors_status))
```

Comment:
This code seems self-documenting to me; I don't find the comments necessary.

```suggestion

    detector_options = ["on", "triggered"]
    detectors_bits = np.flip(np.unpackbits(bin_arr[19:20].view(dtype="u1")))[:3]
    detectors_status = [detector_options[bit] for bit in detectors_bits]
    detectors = dict(zip(["HXM1", "HMX2", "SGM"], detectors_status))
```